app/domain/rotation/rotated_variable.rb

```diff
@@ -39,7 +39,6 @@ def name
     # and returns everything up to th first slash
     #
     def prefix
-      puts '@resource_id', @resource_id
       @resource_id.match(%r{(.*)/.*})[1]
```

Comment:
Wouldn't these Id utilities be more appropriate on the `Resource` class or even a new `Id` helper class? Would you mind creating a ticket for this if you don't want to address it now?

Reply:
You are 100% right, in that this logic isn't specific to a rotated variable, and in fact different versions of this code exist all over the code base.  But because I didn't want to open that can of worms, and I still don't, I left it like this.  I will make a note to open an issue on this, and the related bits of code.  This is a larger issue than it seems because it opens up questions about how to share not only across features but even across gh projects which our team has never answered in a standardized way.